src/main/java/seedu/address/model/group/Group.java

```diff
@@ -34,7 +34,9 @@ public class Group implements Result {
     /**
      * Message to indicate the constraints for group names.
      */
-    public static final String MESSAGE_CONSTRAINTS = "Group names must be non-empty and alphanumeric";
+    public static final String MESSAGE_CONSTRAINTS =
+            "Group names should only contain alphanumeric characters and spaces, and it should not be blank."
+                    + "\nSpecial characters like -, \', and / are supported.";
```

Comment:
Rephrase to match the user guide

```suggestion
    public static final String MESSAGE_CONSTRAINTS =
            "Group name must not be blank, and it must contain only alphanumeric characters, spaces,"
                    + " slashes, and dashes.";
```